parlai/core/params.py

```diff
@@ -739,10 +739,10 @@ def add_parlai_args(self, args=None):
             help='Use dynamic batching',
         )
         parlai.add_argument(
+            '-v',
             '--verbose',
             dest='verbose',
-            type='bool',
-            default=False,
+            action='store_true',
```

Comment:
Leave as bool. `-v true`

Reply:
I had the feeling from the groupchat people prefer the shortest `-v`?

Reply:
Okay then it should become an exception in the opt loading so it isn't saved/loaded from opts.